tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -14,6 +14,7 @@ source "${kubernetes_dir}/../../gha-run-k8s-common.sh"
 source "${kubernetes_dir}/../../../ci/lib.sh"
 
 KATA_HYPERVISOR="${KATA_HYPERVISOR:-qemu}"
+KBS_INGRESS="${KBS_INGRESS:-}"
```

Comment:
It'd be nice to explain the change a bit more in the description. It took me a while to understand why are you re-inventing the wheel by adding support for nodeport from upstream trustee when nodeport is already supported.

I'd mention something like `To ensure our deployment is as close to upstream trustee replace the custom nodeport handling and replace it with nodeport kustomized flavour from the trustee project.`

(let me emphasize `Use the body to explain what and why you have done something. In most cases, you can leave out details about how a change has been made.` from https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

Reply:
@ldoktor changes applied

Reply:
Hi @GabyCT !

The `KBS_INGRESS` variable is no longer used; you can delete that line.

Reply:
@wainersm mmm https://github.com/kata-containers/kata-containers/blob/461b6e7c93e37f26fc670ea10560ee7c22545a3b/.github/workflows/run-kata-coco-tests.yaml#L47 seems that is still part of the yaml

Reply:
@wainersm changes applied